src/effects/decoder/prev/v1/effects_apply.c

```
@@ -0,0 +1,540 @@
+	}
+
+	//--------------------------------------------------------------------------
+	// read labels
+	//--------------------------------------------------------------------------
+
+	for(ushort i = 0; i < lbl_count; i++) {
+		LabelID l;
+		fread_assert(&l, sizeof(LabelID), stream);
+		Schema *s = GraphContext_GetSchemaByID(gc, l, SCHEMA_NODE);
+		ASSERT(s != NULL);
+		lbl[i] = Schema_GetName(s);
```

cppcheck (2.10-2)

[error] Common realloc mistake

(memleakOnRealloc)